diff -urN a/internal/version/BUILD.bazel b/internal/version/BUILD.bazel
--- b/internal/version/BUILD.bazel	2019-08-14 15:03:13.517521567 -0700
+++ a/internal/version/BUILD.bazel	2019-08-14 15:02:50.109793107 -0700
@@ -4,7 +4,10 @@
     name = "go_default_library",
     srcs = ["version.go"],
     importpath = "cloud.google.com/go/internal/version",
-    visibility = ["//:__subpackages__"],
+    visibility = [
+        "//:__subpackages__",
+        "@com_google_cloud_go_logging//:__subpackages__",
+    ],
 )
 
 go_test(
